hikari/cli.py

```diff
@@ -43,3 +44,13 @@ def main() -> None:
     sys.stderr.write(f"located at {path}\n")
     sys.stderr.write(f"{py_impl} {py_ver} {py_compiler}\n")
     sys.stderr.write(" ".join(frag.strip() for frag in platform.uname() if frag and frag.strip()) + "\n")
+
+    if extensions := importlib.metadata.entry_points().get("hikari"):
+        sys.stderr.write("\nInstalled extensions:\n")
+
+        for extension in extensions:
+            module = extension.load()
+            module_path = os.path.abspath(os.path.dirname(module.__file__))
```

Comment:
I would've used pathlib here personally lol but doesn't matter either eay

Reply:
I copy pasted this from above, but switched both to make use of pathlib